src/screens/HomePage/HomePage.tsx

```diff
@@ -75,14 +75,14 @@ export default function HomePage() {
     location.hash === '#explore-hospitalizations';
   const risingHospitalizationStates = [
     'DE',
+    'FL',
     'IL',
     'MD',
     'MI',
-    'MN',
-    'NJ',
+    'NH',
```

Comment:
Note MN is still rising but it was crowding out PR which feels more important to highlight so I swapped in NH instead.